projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -627,7 +628,13 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy {
     @notifyChanges()
     @WatchColumnChanges()
     @Input()
-    formatter: (value: any) => any;
+    set formatter (value: any) {
+        this._formatter = value;
+        this.grid.summaryService.clearSummaryCache();
```

Comment:
Clearing the summary cache is overkill since we only need somehow to trigger the change detection of the summary cell.